azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -396,6 +397,26 @@
     @Beta(Beta.SinceVersion.V1_2_0)
     String bootDiagnosticsStorageUri();
 
+    /**
+     * @return true if managed service identity is enabled for the virtual machine
```

Comment:
so here, we do have some proper nouns that we need to capitalize in javadocs:
"Managed Service Identity" is an MS service name
"Active Directory" is also an MS product
so pls scrub the javadocs with these